.travis.yml

```diff
@@ -14,15 +14,12 @@ sudo: false
 # to alert us to any real upcoming change in the language that would be
 # incompatible with our existing code.
 #
-# We may choose to test additional versions as time allows,
-# as long as the number of entries in the build matrix doesn't exceed ten.
-# These additional versions will not be required, so they're advisory only.
-# Since Travis runs five jobs at a time, a limit of ten jobs makes sense.
+# We used to test additional versions beyond the last two,
+# but since Travis only runs five jobs at a time throughout all of Exercism,
+# it is better citizenship to not test the additional versions.
 go:
-  - 1.7
-  - 1.6.3
```

Comment:
Hmm, before the change I'm proposing, I would have kept it and just made it optional. We discover in https://github.com/exercism/x-common/issues/539 that the entire Exercism organisation has five concurrent jobs though, so I'm a little doubtful we should keep it, especially if (it being optional) we never care about whether it breaks anyway.

The Go release policy at https://golang.org/doc/devel/release.html is that because 1.8 is released, it is the only supported release, but 1.7 will still receive security fixes. So probably 1.6 will get no support at all.

Should we keep anyway? Having extra build entries is not **too** bad since each of ours only takes a minute or so, and there can be an argument for doing so if there are still users of it.

Now is a chance for us to decide a policy!